apps/ui/public/locales/fi/application.json

```diff
@@ -165,31 +166,37 @@
     "homeCity": "Kotipaikka",
     "heading": "4. Tarkista ja lähetä hakemus",
     "text": "Yhteenveto hakemuksestasi",
-    "basicInfoSubHeading": "Varaajan tiedot",
-    "firstName": "Yhteyshenkilön etunimi",
-    "lastName": "Yhteyshenkilön sukunimi",
-    "applicantTypeLabel": "Hakijan tyyppi",
-    "email": "Sähköpostiosoite",
-    "phoneNumber": "Puhelinnumero",
+    "basicInfoSubHeading": "Hakijan tiedot",
+    "contactPerson": "Yhteyshenkilö",
+    "contactInfo": "Yhteystiedot",
+    "address": "Postiosoite",
     "additionalInformation": "Lisätietoja varauksen käyttötarkoituksesta",
     "organisation": {
-      "name": "Hakijan nimi",
-      "coreBusiness": "Ydintoiminta"
+      "name": "Yhdistyksen tai yhteisön virallinen nimi",
+      "coreBusiness": "Ydintoiminta",
+      "registrationNumber": "Rekisterinumero"
     },
     "applicationEvent": {
-      "name": "Kausivarauksen nimi",
+      "applicationInfo": "Hakemuksen tiedot",
       "numPersons": "Ryhmän koko",
       "ageGroup": "Ikäryhmä",
       "abilityGroup": "Tasoryhmä",
       "purpose": "Varauksen käyttötarkoitus",
       "additionalInfo": "Lisätietoja varaukseen liittyen",
-      "begin": "Kauden aloituspäivä",
-      "end": "Kauden päätöspäivä",
-      "minDuration": "Varauksen vähimmäiskesto",
-      "maxDuration": "Varauksen enimmäiskesto",
-      "eventsPerWeek": "Varausten määrä viikossa"
+      "period": "Kausi",
+      "duration": "Varauksen kesto",
+      "eventsPerWeek": "Varausten määrä viikossa",
+      "appliedSpaces": "Tilatoiveet",
+      "schedules": "Aikatoiveet",
+      "scheduleTooltip": "Toissijaiset aikatoiveet merkitty sulkumerkeillä ( )",
+      "status": {
+        "UNALLOCATED": "Vastaanotettu",
+        "IN_ALLOCATION": "Käsittelyssä",
+        "HANDLED": "Hyväksytty",
+        "REJECTED": "Hylätty"
+      }
```

Comment:
enum translations on the top level of the json with a key that matches the enum type name